packages/taquito-contracts-library/test/taquito-contracts-library.spec.ts

```diff
@@ -90,19 +101,36 @@ describe('ContractsLibrary tests', () => {
           script,
           entrypoints,
         },
+    
       });
     } catch (e: any) {
+      expect
+      expect(validateAddress(contractAddress)).toEqual(0)
```

Comment:
validateAddress is a function from the '@taquito/utils'. I don't think it should be tested here.

Reply:
removed the line